### PR TITLE
feat: Supports marking a job as failed after specifying a few retries…

### PR DESCRIPTION
**Pull request by wujiuye**
… #361

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ controllerNamespace: ""
 
 # "{\"nat\": \"true\"}"
 jobNodeSelector: ""
+jobBackoffLimit:
```
Comment (chivalryq) [inline chart/values.yaml:15]: A default value?

**Review by chivalryq — APPROVED** [review]
This feature is generally LGTM. We can consider make it a configuration level properties.

**Comment by chivalryq** [comment]
Also please sign the DCO by `git commit --signoff`

File: chart/values.yaml (hunk 1 above)
Comment (wujiuye) [reply chart/values.yaml:15]: The default is the maximum value of int, compatible with the original default value

Comment (chivalryq) [reply chart/values.yaml:15]: How about "" instead of just leaving it empty.

Comment (wujiuye) [reply chart/values.yaml:15]: I agree with your point of view.

**Pull request merged**